Sources/PackageGraph/BoundVersion.swift

```diff
@@ -38,7 +38,7 @@ extension BoundVersion: CustomStringConvertible {
             return version.description
         case .unversioned:
             return "unversioned"
-        case .revision(let identifier):
+        case .revision(let identifier, _):
```

Comment:
Intentionally ignoring the `branch` value here since it doesn't add any additional information.